modules/load-balancer/output.tf

```diff
@@ -0,0 +1,34 @@
+#####################################################
+# Load Balancer Resources
+# Copyright 2020 IBM
+#####################################################
+
+output "lb_id" {
+  description = "The ID of the Load balancer"
+  value       = ibm_is_lb.lbs.*.id
+}
+
+output "lb_pools" {
```

Comment:
Here the o/p is list of all pools with all attributes...

Can we list the id of the respective resources...?